### PR TITLE
Update less_than.cpp

### PR DESCRIPTION
**Pull request by MonarchChakri**

### CODE REVIEW

File: src/operators/less_than.cpp [hunk 1]
```diff
@@ -31,6 +31,48 @@
 #include "../bigint.hpp"
 
 namespace libbig {
+  bool largeInt:: operator < (const largeInt& z) {
+    // sign = 0: positive number
+    // sign = 1: negative number
+    // flip if its reverse
 
+    int NEGATIVE = 1;
+    int POSITIVE = 0;
```
Comment (Antimeuh2433) [inline src/operators/less_than.cpp:40]: It's the opposite. If `this->sign` (or, in other words, if `this->sign == 1`), the number is positive. This is stated in [src/operators/ostream_operator.cpp](https://github.com/ayaankhan98/bigINT/blob/main/src/operators/ostream_operator.cpp) : `if (!z.sign) { out << '-'; }`.
```suggestion
    // sign = 0: negative number
    // sign = 1: positive number
    // flip if its reverse

    int POSITIVE = 1;
    int NEGATIVE= 0;
```

File: src/operators/less_than.cpp [hunk 2]
```diff
@@ -31,6 +31,48 @@
 #include "../bigint.hpp"
 
 namespace libbig {
+  bool largeInt:: operator < (const largeInt& z) {
+    // sign = 0: positive number
+    // sign = 1: negative number
+    // flip if its reverse
 
+    int NEGATIVE = 1;
```
Comment (ayaankhan98) [inline src/operators/less_than.cpp:39]: `int` will take 4 bytes, use bool it will take less space

File: src/operators/less_than.cpp (hunk 1 above)
Comment (ayaankhan98) [inline src/operators/less_than.cpp:40]: `int` will take 4 bytes, use bool it will take less space

**Comment by MonarchChakri** [comment]
All the mentioned changes have been done.
Please review.

**Comment by MonarchChakri** [comment]
@ayaankhan98
@Antimeuh2433

**Comment by ayaankhan98** [comment]
@MonarchChakri project structure is reconfigured and automated test and build process is also enabled now, please resolve the merge conflicts

**Comment by MonarchChakri** [comment]
Will close this PR and create a new PR for a cleaner merge

**Comment by MonarchChakri** [comment]
@ayaankhan98

**Pull request closed without merging**